docs/testing.md

```diff
@@ -123,7 +222,7 @@ To federate resources across the clusters [kubefedctl] tool must be installed on
 At any time you can run a cleanup command that will remove kind resources.
 
 ```bash
-make e2e status=clean
+make ci e2e status=clean
 ```
 
 You can do full docker cleanup, but it will force all the docker images to be removed and invalidate the local docker cache. 
```

Comment:
here a section about how to deploy using the operator instead of helm

Reply:
Since this was a small part, I did it here itself.